[Bug]: Linux 下 make dev-app 报错

### 相关组件

其他

### Bug 描述

未检测到 Air，使用普通模式启动时，make dev-app 出现错误

### 期望行为

正常编译并启动

### 相关日志

```shell
/usr/bin/ld: Error: unable to disambiguate: -no_warn_duplicate_libraries (did you mean --no_warn_duplicate_libraries ?)
collect2: error: ld returned 1 exit statu
```

### 操作系统

Debian 12

### 确认事项

- [x] 我已经搜索了现有的 issues，确认这是一个新问题

--- a/scripts/dev.sh
+++ b/scripts/dev.sh
@@ -271,7 +271,9 @@ start_app() {
     log_info "数据库地址: $DB_HOST:${DB_PORT:-5432}"
     
     export CGO_CFLAGS="-Wno-deprecated-declarations -Wno-gnu-folding-constant"
-    export CGO_LDFLAGS="-Wl,-no_warn_duplicate_libraries"
+    if [[ "$(uname)" == "Darwin" ]]; then
+      export CGO_LDFLAGS="-Wl,-no_warn_duplicate_libraries"
+    fi
 
     # 检查是否安装了 Air（热重载工具）
     if command -v air &> /dev/null; then

